dist/main.cjs

```diff
@@ -15054,20 +15064,27 @@ async function main(appId2, privateKey2, repository2, core2, createAppAuth2, req
     type: "app"
   });
   const { data: installation } = await request2(
-    "GET /repos/{owner}/{repo}/installation",
+    "GET /orgs/{org}/installation",
     {
-      owner,
-      repo,
+      org,
```

Comment:
`org` is always empty here if owner2 is passed to this function

Reply:
The `dist/*` files are compiled build files. Better comment on the `main.js` and `lib/main.js`

I think that my PR here addresses your concern https://github.com/timreimherr/create-github-app-token/pull/1

Reply:
This was an issue I ran into when trying to use your fork as an action. It might be that the dist file wasn't up to date at the time I made the comment, I just assumed it would be.

Reply:
ah makes sense, thank you for clarifying